Fix section numbers in man page cross-references. With minor changes, this
patch dates all the way back to arpwatch-2.1a4-man.patch, from RHBZ#15442.

diff -Naur arpwatch-3.1-original/arpsnmp.8.in arpwatch-3.1/arpsnmp.8.in
--- arpwatch-3.1-original/arpsnmp.8.in	2019-12-01 14:01:07.000000000 -0500
+++ arpwatch-3.1/arpsnmp.8.in	2020-11-05 15:13:01.296113145 -0500
@@ -45,7 +45,7 @@
 and reports certain changes via email.
 .Nm
 reads information from a file (usually generated by
-.Xr snmpwalk 3 ) .
+.Xr snmpwalk 1 ) .
 .Pp
 The format of the input file is the same as
 .Ar arp.dat ;
@@ -119,9 +119,9 @@
 .Pp
 .Sh "REPORT MESSAGES"
 See the
-.Xr arpwatch 1
+.Xr arpwatch 8
 man page for details on the report messages generated by
-.Xr arpsnmp 1 .
+.Xr arpsnmp 8 .
 .Sh FILES
 .Bl -tag -width ".Pa /usr/local/arpwatch" -compact
 .It Pa /usr/local/arpwatch
@@ -132,7 +132,7 @@
 vendor ethernet block list
 .Sh "SEE ALSO"
 .Xr arpwatch 8 ,
-.Xr snmpwalk 8 ,
+.Xr snmpwalk 1 ,
 .Xr arp 8 ,
 .Sh AUTHORS
 .An Craig Leres
diff -Naur arpwatch-3.1-original/arpwatch.8.in arpwatch-3.1/arpwatch.8.in
--- arpwatch-3.1-original/arpwatch.8.in	2019-12-01 14:01:07.000000000 -0500
+++ arpwatch-3.1/arpwatch.8.in	2020-11-05 15:14:12.117564292 -0500
@@ -117,9 +117,9 @@
 .Fl r
 flag is used to specify a savefile
 (perhaps created by
-.Xr tcpdump 1
+.Xr tcpdump 8
 or
-.Xr pcapture 1 )
+.Xr pcapture 8 )
 to read from instead
 of reading from the network. In this case
 .Nm
@@ -163,9 +163,9 @@
 .Pp
 .Sh "REPORT MESSAGES"
 Here's a quick list of the report messages generated by
-.Xr arpwatch 1
+.Xr arpwatch 8
 (and
-.Xr arpsnmp 1 ) :
+.Xr arpsnmp 8 ) :
 .Pp
 .Bl -tag -width xxx
 .It Ic "new activity"
@@ -216,9 +216,9 @@
 .Sh "SEE ALSO"
 .Xr arpsnmp 8 ,
 .Xr arp 8 ,
-.Xr bpf 4 ,
-.Xr tcpdump 1 ,
-.Xr pcapture 1 ,
+.Xr bpf 2 ,
+.Xr tcpdump 8 ,
+.Xr pcapture 8 ,
 .Xr pcap 3
 .Sh AUTHORS
 .An Craig Leres
